server/devdocs/bin/generate-components-usage-stats.js

```diff
@@ -24,15 +24,11 @@ var async = require( 'async' ),
 	};
 
 function main() {
-	// extract list of files to index and remove leading ./'s
 	const outFilePath = 'server/devdocs/components-usage-stats.json';
-	const fileList = globby.sync( process.argv.slice( 2 ) )
-		.map( function( fileWithPath ) {
```

Comment:
Since the command that invokes this script no longer pipes the output of `find`, removing the initial `./` is no longer needed. This is the only actual code change of this PR.